CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -34,6 +34,7 @@ function PlayerHospital:PlayerHospital(world, avail_rooms, name)
   self:Hospital(world, avail_rooms, name)
   -- The player hospital in single player can access the Cheat System should they wish to.
   self.hosp_cheats = Cheats(self)
+  self.active_cheats = {} -- records any active toggle-based cheats
```

Comment:
@Alberth289346 I think this one might need to go into `hospital` rather than `playerhospital` and `aihospital` separately?